src/Footer.js

```diff
@@ -0,0 +1,20 @@
+import React from "react";
+
+const Footer = () => {
+  let address = [
+    "123 Fake Street, London, E1 4UD",
+    "hello@fakehotel.com",
+    "0123 456789"
+  ];
+  return (
+    <div>
+      <ul>
+        {address.map(item => {
+          return <li>{item}</li>;
```

Comment:
When doing `.map` to return JSX, you need to add a `key` property to the parent element as this is needed for React to know when it should update if the array changes. e.g.
```
{address.map(item => return <li key={item}>{item}</li>)}
```